src/styles/class-overrides.scss

```diff
@@ -44,11 +44,13 @@
 }
 
 @mixin link-size {
-  border-bottom-width: units('1px') * 3;
+  // border-bottom-width: units('1px') * 3;
+  border-bottom-width: units('05');
 }
 
 @mixin link-size-expanded {
-  border-bottom-width: units('1px') * 10;
+  // border-bottom-width: units('1px') * 10;
+  border-bottom-width: units('105');
```

Comment:
Thanks for solving the conflicts! I took this directly from the Figma and there were no conflicts in the branch I was working in.

Reply:
These weren't conflicts - I was just trying to use the USWDS units. So they are slightly different sizes than in the figma

Reply:
I was referring to the overlap you mentioned earlier.